packages/components/src/ActionBar/ActionBar.stories.js

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
```

Comment:
this looks weid to me to import package not specified in devDependencies 🤔

Reply:
I can live with an exclusion from eslint at this point